host/src/CoreToolsHost/LocalSettingsJsonParser.cs

```diff
@@ -23,7 +23,7 @@ internal static async Task<JsonDocument> GetLocalSettingsJsonAsJObjectAsync()
                 }
                 if (!string.IsNullOrEmpty(fileContent))
                 {
-                    var localSettingsJObject = JsonDocument.Parse(fileContent);
+                    var localSettingsJObject = JsonDocument.Parse(fileContent, new JsonDocumentOptions { CommentHandling = JsonCommentHandling.Skip });
```

Comment:
can we get AllowTrailingCommas = true on this too to match prior behaviour plz? @mattchenderson